### PR TITLE
chore: add new APIs 

### PR DESCRIPTION
**Pull request by poppoerika**
This combines the following PRs:

- https://github.com/momentohq/public-dev-docs/pull/125
- https://github.com/momentohq/public-dev-docs/pull/126
- https://github.com/momentohq/public-dev-docs/pull/127
- https://github.com/momentohq/public-dev-docs/pull/129
- https://github.com/momentohq/public-dev-docs/pull/130

Tickets:

- https://github.com/momentohq/dev-eco-issue-tracker/issues/208
- https://github.com/momentohq/dev-eco-issue-tracker/issues/209
- https://github.com/momentohq/dev-eco-issue-tracker/issues/210
- https://github.com/momentohq/dev-eco-issue-tracker/issues/211
- https://github.com/momentohq/dev-eco-issue-tracker/issues/212
- https://github.com/momentohq/dev-eco-issue-tracker/issues/213 - this is already added to the doc so closed the ticket.

I addressed all the feedback received.
@allenheltondev - For [this](https://github.com/momentohq/public-dev-docs/pull/130#discussion_r1135737362), maybe we can make a ticket to follow up on your feedback.

### CODE REVIEW

**Comment by poppoerika** [comment]
I'm going to close all the above separate PRs.

**Comment by poppoerika** [comment]
I still need to address https://github.com/momentohq/public-dev-docs/pull/127#discussion_r1135743129

**Comment by poppoerika** [comment]
> I still need to address [#127 (comment)](https://github.com/momentohq/public-dev-docs/pull/127#discussion_r1135743129)

This is addressed.

File: docs/develop/api-reference/set-collections.md [hunk 1]
```diff
@@ -117,4 +117,50 @@ Removes multiple elements from an existing set item.
...
+</details>
+
+### SetContainsElements
+Checks if provided elements are in the given set.
+
+| Name      | Type       | Description                         |
+|-----------|------------|-------------------------------------|
+| cacheName | String     | Name of the cache.                  |
+| setName   | String     | Name of the set item. |
+| elements  | String[] \ | bytes[]                             | Array of element names to check existence of.   |
```
Comment (cprice404) [inline docs/develop/api-reference/set-collections.md]:
```suggestion
| elements  | String[] \ | bytes                             | Array of element names to check existence of.   |
```

**Review by cprice404 — COMMENTED** [review]
left one suggestion but otherwise lgtm

**Review by NoSQLKnowHow — APPROVED** [review]
As long as all of these APIs are in the prod cells, I am good with merging this.

**Comment by poppoerika** [comment]
It looks like `keyExists` hasn't been deployed to prod so will hold off on merging this PR until it's deployed to prod.

**Pull request merged**